airflow/providers/google/cloud/transfers/sql_to_gcs.py

```diff
@@ -103,6 +104,7 @@ def __init__(
         gcp_conn_id: str = 'google_cloud_default',
         delegate_to: Optional[str] = None,
         impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
+        exclude_columns: List[str] = [],
```

Comment:
Please do not use `[]` as default argument. Use `None` and in constructor do `exclude_columns = exclude_columns or []` as we do in many places.

https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/